db/data/my.bookshop-Contents.csv

```diff
@@ -0,0 +1,64 @@
+ID;parent_ID;name;page;book_ID
+f846b0b9-01d4-4f6d-82a4-d79204f62514;;Foreword;3;9b084139-0b1e-43b6-b12a-7b3669d75f02
```

Comment:
Also here would be nice, if UUIDs would not look so similar everywhere, but that is optional.